code/modules/admin/buildmode.dm

```diff
@@ -234,6 +250,20 @@
 			generator_path = type
 			cornerA = null
 			cornerB = null
+		if(AREAEDIT_BUILDMODE)
+			var/target_path = input(user,"Enter typepath:", "Typepath", "/area")
+			var/areatype = text2path(target_path)
+			if(ispath(areatype,/area))
+				var/areaname = input(user,"Enter area name:", "Area name", "Area")
+				if(!areaname || !length(areaname))
+					return
+				storedarea = new areatype
+				storedarea.power_equip = 0
+				storedarea.power_light = 0
+				storedarea.power_environ = 0
+				storedarea.always_unpowered = 0
+				storedarea.name = areaname
+				areaimage.loc = storedarea
 
```

Comment:
The reason for setting all these things to 0 is so that you can use VV to do that by hand, right?